tests/functional/u/useless/useless_super_delegation.py

```diff
@@ -303,3 +303,13 @@ def __str__(self):
 
     def __hash__(self): # [useless-super-delegation]
         return super().__hash__()
+
+
+# Reported in https://github.com/PyCQA/pylint/issues/2270
+class Super:
+    def __init__(self, *args):
+        self.args = args
+
+class Sub(Super):
+    def __init__(self, a, b):
+        super().__init__(a, b)
```

Comment:
How about when the first argument is required? I see the useless-super-delegation for this:

```python
class Super:
    def __init__(self, *args):
        self.args = args

class Sub(Super):
    def __init__(self, a, *args):
        super().__init__(a, *args)
```

Reply:
So you would want this to raise whenever there is only a variadic in `super()` and one or more positional in child?

Reply:
@DanielNoord that's exactly it. It's just a variation of the original issue which someone may encounter.

Reply:
I think that will work @DanielNoord. I haven't time and haven't played with other variations but I think it's good to go. Thanks!

Reply:
Updated!